### PR TITLE
Changes for category based home page

### PR DESCRIPTION
**Pull request by chrisrrowland**
Still no global theme

### CODE REVIEW

File: ccm_web/client/src/models/feature.ts [hunk 1]
```diff
@@ -13,10 +13,17 @@ const mergeSectionProps: FeatureDataProps = {
...
+  id: 'CanvasGradebookTools',
   title: 'Gradebook Tools',
-  description: 'Trim the gradebook from Canvas, or trim the gradebook from a third party to correct format',
+  description: 'Trim the gradebook from Canvas',
+  ordinality: 2
+}
+
+const thirdPartygradebookToolsProps: FeatureDataProps = {
+  id: 'ThirdPartyGradebookTools',
+  title: 'Gradebook Tools',
```
Comment (pushyamig) [inline ccm_web/client/src/models/feature.ts:25]: The title can be changed  to as per the UX [mockups](https://github.com/tl-its-umich-edu/UX-Work/blob/master/ccm/wireframes/gradebook_feature/gradebook_sc_instructor.pdf). But I want to confirm from with melinda on that

Comment (pushyamig) [reply ccm_web/client/src/models/feature.ts:25]: instead of `route: '/gradebook-other'` it can be names as `/gradebook-thirdparty`

**Comment by pushyamig** [comment]
This looks good, I have few question on naming of gradebook tools if that clarified we can merge it

File: ccm_web/client/src/models/feature.ts (hunk 1 above)
Comment (pushyamig) [reply ccm_web/client/src/models/feature.ts:25]: @chrisrrowland can you change the title and route as per the mockup. I will approve once done

Comment (chrisrrowland) [reply ccm_web/client/src/models/feature.ts]: done

File: ccm_web/client/src/models/FeatureUIData.tsx [hunk 2]
```diff
@@ -22,11 +30,18 @@ const mergeSectionCardProps: FeatureUIProps = {
...
   component: MergeSections,
-  route: '/gradebook'
+  route: '/gradebook-canvas'
+}
+
+const thirdPartyGradebookCardProps: FeatureUIProps = {
+  data: thirdPartygradebookToolsProps,
+  icon: <PostAddOutlinedIcon fontSize='large' />,
+  component: MergeSections,
+  route: '/gradebook-other'
```
Comment (pushyamig) [inline ccm_web/client/src/models/FeatureUIData.tsx:44]:
```suggestion
  route: '/gradebook-thirdparty'
```

Comment (chrisrrowland) [reply ccm_web/client/src/models/FeatureUIData.tsx]: sorry didn't save that file

Comment (chrisrrowland) [reply ccm_web/client/src/models/FeatureUIData.tsx]: though its could just as well be named /aasdfwaefqwefasdf

**Review by pushyamig — APPROVED** [review]
This looks Good. Dan has some suggestion about how the Gradebook tools cards should have a explanatory text changed. But We did not receive input from him yet. I will merge it and will create an issue to fix it

**Pull request merged**